README.md

```diff
@@ -22,6 +24,7 @@ set the `GITHUB_TOKEN` environment variable:
 uses: rokroskar/workflow-run-cleanup-action
 env:
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  DEBUG: TRUE
```

Comment:
maybe we shouldn't have the DEBUG option here set to TRUE as an example? People usually copy/paste these lines to quickly get started and don't pay much attention...

Reply:
done.